package.json

```diff
@@ -89,12 +89,12 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
-      "packages/web-components/typescript",
-      "packages/web-components/typescript/**",
-      "packages/web-components/ts-loader",
-      "packages/web-components/ts-loader/**",
+      "packages/web-components/@microsoft/api-extractor",
+      "packages/web-components/@microsoft/api-extractor/**",
+      "packages/web-components/@microsoft/eslint-config-fast-dna",
```

Comment:
eslint-config-fast-dna also has a dep on typescript, so hopefully not hoisting it will help prevent issues.

ts-loader is tentatively using the same version now and doesn't need nohoist config.